source/integrate-with-integration-environment/integrate-with-code-flow.html.md.erb

```diff
@@ -52,10 +50,22 @@ Host: oidc.integration.account.gov.uk
 
 <%= warning_text('This code example uses formatting that makes it easier to read. If you copy the example, you must make sure the request is:<ul><li>a continuous line of text separating each parameter with an ampersand (&)</li><li>not split across multiple lines</li><li>without any additional separators such as newline, commas or tabs</li></ul>') %>
 
-### Make an authorisation request for authentication and identity
-If you need identity assurance as well as authentication, use this example message to make a `GET` request.
+### Make a request for authentication and identity
+
+If you need to authenticate and process identity data, you should send 2 separate requests, one for authentication and one for identity. 
+
+1. [Send a request to the `/authorize` endpoint to authenticate your user](/integrate-with-integration-environment/integrate-with-code-flow/#make-a-request-to-the-authorize-endpoint) specifying the Vector of Trust (vtr) parameter as ["Cl.Cm"].
+1. Send a request for identity to the `/authorize` endpoint specifying the Vector of Trust parameter `vtr` as ["Cl.Cm.P2"].
```

Comment:
This isn't to sort out here as it's in other parts of the docs already, but I think we should check our capitalisation of 'Vector of Trust'. I think it should be lowercase when talking about it as a parameter and only capitalised for the title of the framework. I'll create an issue on the board!

Reply:
I have added this to the same document for restructuring to update across the whole set of documentation.